domain/BUILD.bazel

```diff
@@ -150,13 +150,6 @@ domain_test(
     deps = TEST_DEPS,
 )
 
-domain_test(
-    name = "OnboardingFlowControllerTest",
```

Comment:
Why was this removed? I believe the test was renamed.

Reply:
Oh - I will change this, then!